x-pack/filebeat/input/gcs/test/mock/data.go

```diff
@@ -4,24 +4,48 @@
 
 package mock
```

Comment:
Can this package be nested back one directory level? The test directory is not really doing any work here.

Reply:
sure

Reply:
@efd6 updated with the recommended fixes